### PR TITLE
Print no target

### LINKED ISSUE #19: `dg deploy` quits silently if there's no target to deploy
It should show something to let user know there's no target to build or deploy.

### PR DESCRIPTION
**Pull request by henteko**
fix #19

### CODE REVIEW

**Comment by henteko** [comment]
@tnj Check please

File: lib/deploygate/commands/deploy/build.rb [hunk 1]
```diff
@@ -141,6 +144,16 @@ def create_provisioning(identifier, uuid)
 
             DeployGate::Builds::Ios::Export.select_profile(provisioning_profiles)
           end
+
+          def print_no_target
+            message = <<EOF
+
+No target.
```
Comment (tnj) [inline lib/deploygate/commands/deploy/build.rb]:
```
No deploy target found.
Please run on the root directory of iOS project or specify .apk/.ipa file to deploy.
```

**Comment by henteko** [comment]
@tnj fix!

**Comment by tnj** [comment]
LGTM

**Comment by henteko** [comment]
thank you!

**Pull request merged**